step3.nims

```diff
@@ -21,6 +21,7 @@ if defined(emscripten):
   --gc:arc # GC:arc is friendlier with crazy platforms.
   --exceptions:goto # Goto exceptions are friendlier with crazy platforms.
   --define:noSignalHandler # Emscripten doesn't support signal handlers.
+  --define:vmathArrayBased # For interop with OpenGL
```

Comment:
`--define:vmathArrayBased` this is not a good idea array based is slower. ObjArray default is like 20% faster. There should be other ways to cast that.